Pipfile

```diff
@@ -15,23 +15,22 @@ decorator = "*"
 click = "==8.1.3"
 
 [dev-packages]
-bandit = "*"
 black = "*"
 boto3-stubs = "*"
 coverage = "*"
 coveralls = "*"
 mock-ssh-server = "*"
 moto = "*"
 mypy = "*"
-pylama = {extras = ["all"], version = "*"}
-pytest = "*"
+pre-commit = "*"
 pydocstyle = "*"
```

Comment:
I don't think we need `pydocstyle` anymore since that's covered by `ruff`

Reply:
Good catch, thanks!